client/packages/invoices/src/Prescriptions/DetailView/PrescriptionLineEdit/PrescriptionLineEdit.tsx

```diff
@@ -201,11 +203,18 @@ export const PrescriptionLineEdit: React.FC<PrescriptionLineEditModalProps> = ({
         />
       }
       okButton={
-        <DialogButton
+        <LoadingButton
           disabled={!currentItem}
-          variant="ok"
+          isLoading={isSaving}
+          startIcon={<CheckIcon/>}
+          loadingStyle={{iconColor:'secondary.main' }}
+          variant="contained"
+          color='secondary'
+          aria-label={t('button.ok')}
           onClick={() => handleSave(onClose)}
-        />
+        >
+          {t('button.ok')}
+          </LoadingButton>
```

Comment:
Did the format on save stop working? The "button.ok" bit should be more indented than the loading button 👀

Reply:
Oh, it picked up 5 lines to reformat. Have done this now!